### PR TITLE
Update azure-arc-enabled-virtual-machines.md

### PR DESCRIPTION
**Pull request by BenoitNgs**
Add precision to install AZCli and use RDP connection to deploy bridge

### CODE REVIEW

**Comment by PRMerger15** [comment]
@BenoitNgs : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by ksurjan — APPROVED** [review]

File: azure-stack/hci/manage/azure-arc-enabled-virtual-machines.md [hunk 1]
```diff
@@ -99,7 +101,7 @@ The following firewall URL exceptions are needed on all servers in the Azure Sta
 
 ## Install PowerShell modules and update extensions
 
-To prepare to install Azure Arc Resource Bridge on an Azure Stack HCI cluster and create a VM cluster-extension, perform these steps:
+To prepare to install Azure Arc Resource Bridge on an Azure Stack HCI cluster and create a VM cluster-extension, perform these steps (throught RDP or console session, remote Powershell is not supported):
```
Comment (sethmanheim) [inline azure-stack/hci/manage/azure-arc-enabled-virtual-machines.md:104]: Typo in this sentence, "throught" is not a word.

File: azure-stack/hci/manage/azure-arc-enabled-virtual-machines.md [hunk 2]
```diff
@@ -60,6 +60,8 @@ Only one Arc Resource Bridge can be deployed on a cluster. Each Azure Stack HCI
 Deploying Azure Arc Resource Bridge requires the following:
 
 - The latest version of Azure CLI installed on all servers of the cluster.
+  - To install Azure CLI on each cluster node use RDP connection.
+  - Follow the instructions on this KB: https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-windows
```
Comment (sethmanheim) [inline azure-stack/hci/manage/azure-arc-enabled-virtual-machines.md:64]: This is an article, not a KB, and the link should be relative, i.e. /cli/azure/install-azure-cli-windows

**Comment by Jak-MS** [comment]
#label:"aq-pr-triaged"

**Comment by PRMerger12** [comment]
@BenoitNgs : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: azure-stack/hci/manage/azure-arc-enabled-virtual-machines.md (hunk 1 above)
Comment (BenoitNgs) [reply azure-stack/hci/manage/azure-arc-enabled-virtual-machines.md]: Sorry, through vs throught.

File: azure-stack/hci/manage/azure-arc-enabled-virtual-machines.md (hunk 2 above)
Comment (BenoitNgs) [reply azure-stack/hci/manage/azure-arc-enabled-virtual-machines.md]: Done, replace KB by article and relative URI.

**Comment by ksurjan** [comment]
#sign-off

File: azure-stack/hci/manage/azure-arc-enabled-virtual-machines.md [hunk 3]
```diff
@@ -99,7 +101,7 @@ The following firewall URL exceptions are needed on all servers in the Azure Sta
 
 ## Install PowerShell modules and update extensions
 
-To prepare to install Azure Arc Resource Bridge on an Azure Stack HCI cluster and create a VM cluster-extension, perform these steps:
+To prepare to install Azure Arc Resource Bridge on an Azure Stack HCI cluster and create a VM cluster-extension, perform these steps (through RDP or console session, remote Powershell is not supported):
```
Comment (Court72) [inline azure-stack/hci/manage/azure-arc-enabled-virtual-machines.md:104]:
```suggestion
To prepare to install Azure Arc Resource Bridge on an Azure Stack HCI cluster and create a VM cluster-extension, perform these steps (through RDP or console session; remote Powershell isn't supported):
```

File: azure-stack/hci/manage/azure-arc-enabled-virtual-machines.md [hunk 4]
```diff
@@ -60,6 +60,8 @@ Only one Arc Resource Bridge can be deployed on a cluster. Each Azure Stack HCI
 Deploying Azure Arc Resource Bridge requires the following:
 
 - The latest version of Azure CLI installed on all servers of the cluster.
+  - To install Azure CLI on each cluster node use RDP connection.
+  - Follow the instructions on this article: [Install Azure CLI](/cli/azure/install-azure-cli-windows).
```
Comment (Court72) [inline azure-stack/hci/manage/azure-arc-enabled-virtual-machines.md:64]:
```suggestion
  - Follow the instructions in [Install Azure CLI](/cli/azure/install-azure-cli-windows).
```

File: azure-stack/hci/manage/azure-arc-enabled-virtual-machines.md [hunk 5]
```diff
@@ -60,6 +60,8 @@ Only one Arc Resource Bridge can be deployed on a cluster. Each Azure Stack HCI
 Deploying Azure Arc Resource Bridge requires the following:
 
 - The latest version of Azure CLI installed on all servers of the cluster.
+  - To install Azure CLI on each cluster node use RDP connection.
```
Comment (Court72) [inline azure-stack/hci/manage/azure-arc-enabled-virtual-machines.md:63]:
```suggestion
  - To install Azure CLI on each cluster node, use RDP connection.
```

**Comment by sethmanheim** [comment]
@BenoitNgs Awesome, thank you. OK to merge.

**Comment by PRMerger7** [comment]
@Court72 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**